### ISSUE TITLE
No files created after deeplabcut.analyze_videos()

### ISSUE THREAD

**Issue by Darkweiss** [DeepLabCut/DeepLabCut#1083]
Thanks for opening this issue, and thanks for using DeepLabCut (we hope you are enjoying it ;). Please fill out the template completely, including the full "traceback" and input code that you ran to hit this error.

**Your Operating system and DeepLabCut version**

Please state your operating system, env, and which version of DeepLabCut you are using.
**Using this Google Colab notebook:** https://colab.research.google.com/github/AlexEMG/DeepLabCut/blob/master/examples/COLAB_YOURDATA_TrainNetwork_VideoAnalysis.ipynb
DeepLabCut Version: 2.1.9

**Describe the problem**
When running deeplabcut.analyze_videos(), no files are created in the process. 

Please place code inside this:

<details><summary>Code output</summary><p>

Using snapshot-140000 for model /content/drive/My Drive/Colab Notebooks/GPU_DLC_Test-Luka-2021-01-13/dlc-models/iteration-0/GPU_DLC_TestJan13-trainset95shuffle1
Initializing ResNet

  0%|          | 0/7197 [00:00<?, ?it/s]Analyzing all the videos in the directory...
Starting to analyze %  /content/drive/My Drive/Colab Notebooks/GPU_DLC_Test-Luka-2021-01-13/JustOne/mouse1.avi
/content/drive/My Drive/Colab Notebooks/GPU_DLC_Test-Luka-2021-01-13/JustOne  already exists!
Loading  /content/drive/My Drive/Colab Notebooks/GPU_DLC_Test-Luka-2021-01-13/JustOne/mouse1.avi
Duration of video [s]:  359.68 , recorded with  20.01 fps!
Overall # of frames:  7197  found with (before cropping) frame dimensions:  1280 1024
Starting to extract posture
The videos are analyzed. Now your research can truly start! 
 You can create labeled videos with 'create_labeled_video'
If the tracking is not satisfactory for some videos, consider expanding the training set. You can use the function 'extract_outlier_frames' to extract a few representative outlier frames.
DLC_resnet50_GPU_DLC_TestJan13shuffle1_140000
</p></details>

**Additional context**
I've tried moving the video in a different directory and it didn't work. I've also tried a shorter .mp4 video and the analysis started on that one normally.

**Comment by Darkweiss** [comment]
Trimming the video on my computer and converting it to mp4 results in the same thing, it always stops on 72 frames:

Config:
{'all_joints': [[0], [1]],
 'all_joints_names': ['Snout', 'Tail base'],
 'batch_size': 1,
 'crop_pad': 0,
 'dataset': 'training-datasets/iteration-0/UnaugmentedDataSet_GPU_DLC_TestJan13/GPU_DLC_Test_Luka95shuffle1.mat',
 'dataset_type': 'imgaug',
 'deterministic': False,
 'fg_fraction': 0.25,
 'global_scale': 0.8,
 'init_weights': '/usr/local/lib/python3.6/dist-packages/deeplabcut/pose_estimation_tensorflow/models/pretrained/resnet_v1_50.ckpt',
 'intermediate_supervision': False,
 'intermediate_supervision_layer': 12,
 'location_refinement': True,
 'locref_huber_loss': True,
 'locref_loss_weight': 1.0,
 'locref_stdev': 7.2801,
 'log_dir': 'log',
 'mean_pixel': [123.68, 116.779, 103.939],
 'mirror': False,
 'net_type': 'resnet_50',
 'num_joints': 2,
 'optimizer': 'sgd',
 'pairwise_huber_loss': True,
 'pairwise_predict': False,
 'partaffinityfield_predict': False,
 'regularize': False,
 'scoremap_dir': 'test',
 'shuffle': True,
 'snapshot_prefix': '/content/drive/My Drive/Colab '
                    'Notebooks/GPU_DLC_Test-Luka-2021-01-13/dlc-models/iteration-0/GPU_DLC_TestJan13-trainset95shuffle1/test/snapshot',
 'stride': 8.0,
 'weigh_negatives': False,
 'weigh_only_present_joints': False,
 'weigh_part_predictions': False,
 'weight_decay': 0.0001}
Using snapshot-140000 for model /content/drive/My Drive/Colab Notebooks/GPU_DLC_Test-Luka-2021-01-13/dlc-models/iteration-0/GPU_DLC_TestJan13-trainset95shuffle1
Initializing ResNet




  0%|          | 0/1807 [00:00<?, ?it/s]Starting to analyze %  /content/drive/MyDrive/Colab Notebooks/GPU_DLC_Test-Luka-2021-01-13/JustOne/ShortVideo.mp4
/content/drive/MyDrive/Colab Notebooks/GPU_DLC_Test-Luka-2021-01-13/JustOne  already exists!
Loading  /content/drive/MyDrive/Colab Notebooks/GPU_DLC_Test-Luka-2021-01-13/JustOne/ShortVideo.mp4
Duration of video [s]:  60.23 , recorded with  30.0 fps!
Overall # of frames:  1807  found with (before cropping) frame dimensions:  1920 1080
Starting to extract posture




  2%|▏         | 36/1807 [00:00<00:08, 205.74it/s]



  3%|▎         | 54/1807 [00:00<00:10, 160.64it/s]



  4%|▍         | 72/1807 [00:00<00:12, 140.15it/s]The videos are analyzed. Now your research can truly start! 
 You can create labeled videos with 'create_labeled_video'
If the tracking is not satisfactory for some videos, consider expanding the training set. You can use the function 'extract_outlier_frames' to extract a few representative outlier frames.
DLC_resnet50_GPU_DLC_TestJan13shuffle1_140000

**Comment by jeylau** [comment]
Hello @Darkweiss, the 72nd frame of your video is likely corrupted, causing the video analysis to stop prematurely.
You could have a look at the following technique to thoroughly check the integrity of your video [https://github.com/DeepLabCut/DeepLabCut/issues/982#issuecomment-723480607](url). Also note that we've recently fixed that behavior with #1058, which will be available soon in our next release 😊

**Comment by Darkweiss** [comment]
That makes total sense. Thanks for the reply!
